### PR TITLE
updating alert for vrops db diskspace

### PR DESCRIPTION
**Pull request by sunnyrarora**
Updated code for getting alert VROpsDiskpaceUsageCritical.
Please review and approve. -Thanks

### CODE REVIEW

File: prometheus-exporters/vrops-exporter/alerts/virtualmachine.alerts [hunk 1]
```diff
@@ -32,5 +30,3 @@ groups:
       meta: "Disk almost full with over 90% for {{ $labels.virtualmachine }}. Please increase disk size."
       playbook: docs/devops/alert/vcenter/
     annotations:
-      description: "Disk almost full with over 90% for {{ $labels.virtualmachine }}. Please increase disk size."
-      summary: "Disk almost full with over 90% for {{ $labels.virtualmachine }}. Please increase disk size."
```
Comment (richardtief) [inline prometheus-exporters/vrops-exporter/alerts/virtualmachine.alerts:36]: Why did you remove this?

File: prometheus-exporters/vrops-exporter/alerts/virtualmachine.alerts [hunk 2]
```diff
@@ -20,9 +20,7 @@ groups:
       summary: "Shadow VM {{ $labels.virtualmachine }} is inaccessible. ({{ $labels.datacenter }}, {{ $labels.vccluster }}, {{ $labels.hostsystem }})"
   - alert: VROpsDiskpaceUsageCritical
     expr: >
-      vrops_virtualmachine_diskspace_virtual_machine_used_gigabytes{virtualmachine =~ "vrops.+"} offset 5m
-      / ignoring (statkey) vrops_virtualmachine_diskspace_gigabytes{virtualmachine =~ "vrops.+"} offset 5m
-      > 0.9
+    vrops_virtualmachine_database_disk_usage_percent{virtualmachine=~"vrops-vc.*"} > 90
```
Comment (richardtief) [inline prometheus-exporters/vrops-exporter/alerts/virtualmachine.alerts:23]: Please use the correct indentation here.

**Review by richardtief — CHANGES_REQUESTED** [review]

File: prometheus-exporters/vrops-exporter/alerts/virtualmachine.alerts (hunk 1 above)
Comment (sunnyrarora) [reply prometheus-exporters/vrops-exporter/alerts/virtualmachine.alerts]: not sure it got left out. Added it again -Thanks

File: prometheus-exporters/vrops-exporter/alerts/virtualmachine.alerts (hunk 2 above)
Comment (sunnyrarora) [reply prometheus-exporters/vrops-exporter/alerts/virtualmachine.alerts]: Done !

**Review by richardtief — APPROVED** [review]
lgmt

**Review by viennaa — APPROVED** [review]
